packages/drivechain_client/lib/service.dart

```diff
@@ -35,7 +44,16 @@ class DrivechainService extends InheritedWidget {
 
   /// Returns a tuple of the confirmed and pending balance in satoshi
   Future<GetBalanceResponse> getBalance() async {
+    const cacheKey = 'balance';
+    if (_balanceCache.containsKey(cacheKey) &&
+        _cacheTimestamps[cacheKey] != null &&
+        DateTime.now().difference(_cacheTimestamps[cacheKey]!) < const Duration(minutes: 1)) {
+      return _balanceCache[cacheKey]!;
+    }
```

Comment:
Hva er tanken med å cache dette?

Reply:
I think it's better if we do caching on the server, tbh